Content.Client/_DV/AACTablet/UI/AACWindow.xaml.cs

```diff
@@ -184,6 +217,14 @@ private static Button CreatePhraseButton(string text, string styleClass)
 
     private void OnPhraseButtonPressed(ProtoId<QuickPhrasePrototype> phraseId)
     {
-        PhraseButtonPressed?.Invoke(phraseId);
+        if (ShouldBuffer.Pressed)
+        {
+            _phraseBuffer.Add(phraseId);
+            UpdateBufferText();
+        }
+        else
+        {
+            PhraseButtonPressed?.Invoke(new List<ProtoId<QuickPhrasePrototype>> {phraseId});
```

Comment:
make another list for this and clear/add/invoke instead of reallocating every time